src/test/java/com/jcabi/log/VerboseProcessTest.java

```diff
@@ -477,4 +568,37 @@ public int decide(final LoggingEvent event) {
             return decision;
         }
     }
+
+    /**
+     * Logger appender that compiles a list of all LoggingEvents.
+     * @author dean.e.clark
+     */
+    private class TestAppender extends AppenderSkeleton {
```

Comment:
@mkordas tried to do so, but then Travis CI complains about the class having a protected method